src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java

```diff
@@ -0,0 +1,29 @@
+package com.nulabinc.backlog4j.api.option;
+
+import com.nulabinc.backlog4j.http.NameValuePair;
+
+/**
+ * Parameters for update status API.
+ *
+ * @author nulab-inc
+ */
+public class UpdateStatusParams extends PatchParams {
```

Comment:
Status color(param: color) should be changeable.
Please refer https://developer.nulab.com/docs/backlog/api/2/update-status/#form-parameters